run_with_software_rendering.sh

```
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Run the app with software rendering enabled
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)